.stylelintrc

```
@@ -1,6 +1,7 @@
 {
   "rules": {
-    "declaration-no-important": true
+    "declaration-no-important": true,
+    "selector-type-no-unknown": null
```

Was not sure if I should do, so looking for insight on this
```suggestion
    "selector-type-no-unknown": [true, { "ignoreTypes": ["-styled-mixin0", "-styled-mixin1"] }]
```

---

Just tried this locally - so this was needed because otherwise we get this Stylelint error:

```
54:3  ✖  Unexpected unknown type selector "-styled-mixin0"   selector-type-no-unknown
```

...which comes from trying to do this:

```jsx
const StyledLink = styled.a`...`;

const StyledSpan = styled.span`
  ${StyledLink}:hover &,
  ${StyledLink}:focus & { ... }
`;
```

...I wonder why this is an issue in Psammead but not Simorgh 🤔 I'm a little concerned about the cause, rather than the symptom.

Otherwise, I'd suggest using a regex so this scales:

```suggestion
    "selector-type-no-unknown": [true, { "ignoreTypes": ["/^-styled-mixin/"] }]
```